python/GaelZllHbb13TeVconfig/general.ini

```diff
@@ -272,6 +280,6 @@ weightF_bTagJES_UP:     <!Weights|weight_noBTag!> * bTagWeight_JESUp
 weightF_bTagJES_DOWN:     <!Weights|weight_noBTag!> * bTagWeight_JESDown
 
 ;(PU)
-weightF_pileUp_UP: <!Weights|GenSign!> * puWeightUp * <!Weights|BTagWeight!> * <!Weights|weightF_EWK!> * <!Weights|weightF_muSF!>
-weightF_pileUp_DOWN: <!Weights|GenSign!> * puWeightDown * <!Weights|BTagWeight!> * <!Weights|weightF_EWK!> * <!Weights|weightF_muSF!>
+weightF_pileUp_UP: <!Weights|weight_noPU!>* puWeightUp
```

Comment:
I think this is the case. 
weight_noPU is defined as 
<!Weights|GenSign!> * **<!Weights|BTagWeight!> *** <!Weights|weightF_muSF!> *** <!Weights|weightF_EWK!>**